samples/hello-world-entity/src/test/java/org/terracotta/management/entity/helloworld/HelloWorldTest.java

```diff
@@ -1,19 +1,19 @@
 /*
  * Copyright Terracotta, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
ohh I saw! I did't do anything. I guess it's the copyright plugin of IntelliJ... Sorry!